Server/ConcordiaCurriculumManager/appsettings.Development.json

```diff
@@ -13,7 +13,7 @@
     "Audience": "ccm-backend"
   },
   "DatabaseSettings": {
-    "ConnectionString": "Host=localhost;Port=5432;Database=ConcordiaCurriculumManager;Username=postgres;Password=password"
```

Comment:
Can you undo this change. It might be easier for you in future development to just set your postgres password to `password`. In psql you can just run `ALTER USER postgres WITH PASSWORD 'password';`, but it's up to you